packages/passport/sdk-sample-app/src/components/zkevm/EthSignTypedDataV4Examples/SeaportCreateListingExample.ts

```diff
@@ -35,7 +35,7 @@ export const getCreateListingPayload = ({ seaportContractAddress, walletAddress,
           identifierOrCriteria: '842809524',
           itemType: '2',
           startAmount: '1',
-          token: '0x114c1f0702547751a546661bc375d95fb2d281a8',
+          token: '0xd46546c8d7ebe9c1b2afe299080eef374618eca2',
```

Comment:
This is now a valid ERC721 contract in testnet meaning we will see the collection image.